### PR TITLE
Create bank transfer gateway for Komoju

### PR DESCRIPTION
**Pull request by camelmasa**
I created bank tranfer gateway for Komoju.
@shioyama Please review if you have time.

### Screen shots

order_r500392717_-_spree_demo_site

order_r500392717_-_spree_demo_site

spree_demo_site_and_slack

### CODE REVIEW

**Comment by shioyama** [comment]
great I'll review now

**Comment by shioyama** [comment]
You need to add `Spree::Gateway::KomojuBankTransfer` to the [engine initializer](https://github.com/camelmasa/spree_komoju/blob/629a916dce49762296ba78fe7684c2b07785e86c/lib/spree_komoju/engine.rb#L8).

**Comment by shioyama** [comment]
One thing that bothers me a bit is that we are hard-coding details of where a bank transfer should be sent to (account number, etc.). This information could change and anyone using an older version of this extension who did not know would have problems. I'm not sure what we can do about that until we have a payment instructions endpoint, but hard-coding it doesn't seem right somehow.

**Comment by camelmasa** [comment]
> You need to add Spree::Gateway::KomojuBankTransfer to the engine initializer.

Thanks I forgot ... I add a commit.

File: app/views/spree/orders/_bank_transfer.html.erb [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<div>
+  Thank you for your order. Instructions have been sent to your e-mail address.<br>
+<br>
+  <h4>Banktransfer ID: <%= source.order_id %></h4>
+  The Banktransfer ID is very important! Please make a note of it and bring it with you when you go to pay at the ATM.<br>
+  If you have chosen to use bank transfer payment, please use a bank transfer form which your bank offers, or please remit at ATM.<br>
+<br>
+  Please make sure to include your banktransfer ID in the payer's name. Please enter your banktransfer ID shown in this email in front of your name as Payer when remitting.<br>
+<br>
```
Comment (shioyama) [inline app/views/spree/orders/_bank_transfer.html.erb]: I think we should end this page here, and end with:

> Please see the confirmation email for further payment instructions.

Comment (camelmasa) [inline app/views/spree/orders/_bank_transfer.html.erb]: Thanks. I think so too. I'll change it.

**Comment by shioyama** [comment]
LGTM :+1:

**Pull request closed without merging**